src/common/rectangle/window.ts

```diff
@@ -5,32 +5,58 @@
 // that can be found in the LICENSE file exposed on Github (readium) in the project repository.
 // ==LICENSE-END==
 
-import { app, Rectangle, screen } from "electron";
-import * as fs from "fs";
-import * as path from "path";
-import { promisify } from "util";
+import * as debug_ from "debug";
+import { BrowserWindow, Rectangle, screen } from "electron";
+import { ConfigRepository } from "readium-desktop/main/db/repository/config";
+import { container } from "readium-desktop/main/di";
+import { debounce } from "readium-desktop/utils/debounce";
 
-const CONFIG_NAME = "windowRectangle.json";
-const PATH = path.join(app.getPath("userData"), CONFIG_NAME);
+// Logger
+const debug = debug_("readium-desktop:common:rectangle");
```

Comment:
Naming suggestion: `readium-desktop:common:windowRectangle` ("rectangle" on its own is vague / generic)